consensus/src/quorum_store/proof_coordinator.rs

```diff
@@ -38,21 +38,24 @@ pub(crate) type ProofReturnChannel = oneshot::Sender<Result<(ProofOfStore, Batch
 struct IncrementalProofState {
     info: SignedDigestInfo,
     aggregated_signature: BTreeMap<PeerId, bls12381::Signature>,
+    aggregated_voting_power: u128,
```

Comment:
do we need to cache this? checking voting power along should be very fast, it's just sum over hashmap lookups

Reply:
@zekun000 - I haven't measured - but:
* we are checking voting power much more often.
* it is an O(n^2) operation currently. going through a hashmap with 70 elements is not trivially cheap, might compare or be more expensive than signature verification, etc.

Reply:
I think it's fine to cache it, but going through a hashmap with 70 elements should be magnitudes cheaper than a signature verification